ui-core/src/main/java/com/adyen/checkout/ui/core/internal/ui/view/AddressFormInput.kt

```diff
@@ -324,6 +332,8 @@ class AddressFormInput @JvmOverloads constructor(
     private fun initHouseNumberInput(styleResId: Int?) {
         styleResId?.let { textInputLayoutHouseNumber?.setLocalizedHintFromStyle(it, localizedContext) }
         editTextHouseNumber?.apply {
+            // TODO
+//            setAutofillHints(View.AUTO)
```

Comment:
There is no autofill hint for house number. Currently, the house number is automatically autofilled into the street field. Should we add logic to extract the number automatically from the street field into the house number field? What do you guys think?

Reply:
We decided to remove autofill hints for this field as there is not much we can do